biogrid/apps/biogrid-mvp/src/app/pages/input-page/input-page.tsx

```diff
@@ -103,20 +105,21 @@ export const InputPage = () => {
           <b>throughout the town</b> so that our buildings can have an immediate
           energy source when they have a spike in energy consumption.
         </p>
-        <div className="startDatePicker">
-          <label>Start Date</label>
-          <DatePicker
-            showPopperArrow={false}
-            selected={startDate}
-            onChange={(date: Date) => setStartDate(date)}
-          />
-        </div>
-        <div className="endDatePicker">
-          <label>End Date</label>
-          <DatePicker
-            showPopperArrow={false}
-            selected={endDate}
-            onChange={(date: Date) => setEndDate(date)}
+        <div className="datePicker">
+          <label>Simulation Date Slider</label>
+          <ReactSlider
+            className="horizontal-slider"
+            onChange={(val) => {
+              const daysFromStart = (val as unknown) as number;
+              const newDate = new Date();
+              newDate.setDate(EARLIEST_DATE.getDate() + daysFromStart);
+              setStartDate(newDate);
+            }}
```

Comment:
Also, does this assume the start time as midnight, or can we have two options for users - a start and end time?
Fine with leaving as is in this PR, but maybe add a TODO for users to be able to select a start time as well?

Reply:
Oh it would be like start day and end day. Currently, the simulation runs the 24 hours of the date chosen #89